### PR TITLE
[Task-1] Setup project using React template

### PR DESCRIPTION
**Pull request by gutakk**
Resolve https://github.com/gutakk/react-surveyor/issues/1

## What happened 👀

- Generate project using React template
- Update `README`
- Add `PULL_REQUEST_TEMPLATE`
- Add `ISSUE_TEMPLATE`
 
## Insight 📝

N/A
 
## Proof Of Work 📹

The application is able to run locally.
Screen Shot 2564-06-03 at 17 59 14

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,71 @@
+{
+  "name": "react-surveyor",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@nimbl3/eslint-config-nimbl3": "2.1.1",
```
Comment (carryall) [inline package.json:6]: @gutakk Regarding the previous comment, the package name here should be changed too 😅 
```suggestion
    "@nimblehq/eslint-config-nimble": "2.1.1",
```

**Review by carryall — COMMENTED** [review]
There are few changes suggested regarding the bug from React template 😅

**Review by rafayet-monon — COMMENTED** [review]
@gutakk Another thing that you will need is absolute import. It's not yet in `react-templates` but it's planned in the next version. Please have a look at [this issue](https://github.com/nimblehq/react-templates/issues/22). There's a link that says how to configure it.

File: .eslintrc.js [hunk 2]
```diff
@@ -0,0 +1,97 @@
+module.exports = {
+  env: {
+    es6: true,
+    browser: true,
+    node: true,
+    jest: true
+  },
+  extends: [
+    '@nimbl3/eslint-config-nimbl3',
```
[the remark this answers is not in the document]
Comment (gutakk) [reply .eslintrc.js]: Fixed in 1719516511676185d618ff1b59033bc05765c0f9

File: package.json (hunk 1 above)
Comment (gutakk) [reply package.json]: Fixed in 1719516511676185d618ff1b59033bc05765c0f9

File: .eslintrc.js [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+  },
+  settings: {
+    react: {
+      version: 'detect'
+    },
+    'import/resolver': {
+      node: {
+        extensions: ['.ts', '.tsx']
+      }
+    }
```
Comment (rafayet-monon) [inline .eslintrc.js:95]: @gutakk I think you can setup the eslint like this -
```suggestion
    },
    "import/resolver": {
      "node": {
        "paths": ["src"]
      }
    }
```

**Comment by gutakk** [comment]
> @gutakk Another thing that you will need is absolute import. It's better to do at the setup. It's not yet in `react-templates` but it's planned in the next version. Please have a look at [this issue](https://github.com/nimblehq/react-templates/issues/22). There's a link that says how to configure it.

Fixed in 0ab6414483b5ace50e385b035348cab51a0166ff

**Review by carryall — APPROVED** [review]

**Review by rafayet-monon — APPROVED** [review]

**Pull request merged**